proto/neutron/dex/query.proto

```diff
@@ -133,10 +133,17 @@ message QueryParamsResponse {
 message QueryGetLimitOrderTrancheUserRequest {
   string address = 1;
   string tranche_key = 2;
+  bool calc_withdrawable_shares = 3;
```

Comment:
i suppose this is not API breaking cause it's just a fiiled addition, but could you confirm pls?

Reply:
Correct. If a value is not supplied it will default to false and the pre-existing behavior will be maintained